.travis.yml

```diff
@@ -7,6 +7,7 @@ services:
 env:
   - CRATE=.
   - CRATE=spaceapi
+  - CRATE=spaceapi_server
 script: cd $CRATE && cargo build --verbose && cargo test --verbose
```

Comment:
It fixes the tests for the iron branch. Previously, tests were not run.

Yes, I could have put this in a separate pull request, as it's independent. But that would have been a bit of overkill :) As long as it's a separate commit I think it's fine.